src/App.js

```diff
@@ -139,6 +140,18 @@ const App = compose(injectState)(({ editing, setEditing, state, effects }) => {
       <Modal />
       <Toast {...toast}>{toast.component}</Toast>
     </div>
+  ));
+
+  return (
+    <ApiContext.Provider
+      value={initializeApi({
+        onUnauthorized: response => {
+          window.location.reload();
+        },
+      })}
+    >
+      <AppWithApi />
+    </ApiContext.Provider>
```

Comment:
It may make sense to put this provider nested inside the `ContextProvider` below...
```
<ContextProvider>
  <ApiContext.Provider value={...}>
    <App />
  </ApiContext.Provider>
</ContextProvider>
```